### PR TITLE
Feature/9 implement redux

### PR DESCRIPTION
**Pull request by georggrubner**
Issue #9

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                            currentDataSetIndex={this.state.currentDataSetIndex}
+                            dataSets={transformDataSets(dataSets)}
+                            handleChangeDataSet={this.handleChangeDataSet}
+                        />
+                    </Col>
+                    <Col mdPush={10}>
+                        <Provider store={store}>
+                            <Visualizer
+                                dataSet={dataSets[this.state.currentDataSetIndex]}
+                            />
```
Comment (danielbartsch) [inline src/components/App.js]: one line

File: src/components/DataSetDropdownControl.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    const onSelect = eventkey => handleChangeDataSet(eventkey)
+    return (
+        <DropdownButton title={dataSets[currentDataSetIndex].name} id={'input'}>
+            {dataSets.map((dataSet, index) => (
+                <MenuItem
+                    key={dataSets[index].index}
+                    eventKey={dataSets[index].index}
+                    onSelect={onSelect}
+                >
+                    {dataSets[index].name}
```
Comment (danielbartsch) [inline src/components/DataSetDropdownControl.js]: `dataSets[index] === dataSet` you're mapping over it

File: src/components/visualizer/controls/UserControls/ProjectionControl.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import styles from './styles.css'
+
+const Projection = ({
+    bsStyle,
+    columns,
+    toggledProjections,
+    onClickToggleColumn,
+}: ProjectionControlProps) => (
+    columns.map((columnValue, index) => (index % 2 === 0 ?
+        (<ButtonGroup
```
Comment (danielbartsch) [inline src/components/visualizer/controls/UserControls/ProjectionControl.js]: weird parenthesis style

```diff
-condition ?
-    (<Component
+condition ? (
+    <Component

File: src/etc/const.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
 // @flow
 export const TOTAL = 'Total'
 export const CONCAT_STRING = ' > '
-export const PROJECTED_DIMENSIONS = 'projectedDimensions'
-export const PROJECTED_METRICS = 'projectedMetrics'
+export const DIMENSIONS = 'dimensions'
+export const METRICS = 'metrics'
+export const OPERATOR = 2
```
Comment (danielbartsch) [inline src/etc/const.js]: operators are `*`, `/`, `+`, `-`, `%`. this is an operand

File: src/components/App.js (hunk 1 above)
Comment (georggrubner) [reply src/components/App.js]: 42bdf62

File: src/components/DataSetDropdownControl.js (hunk 2 above)
Comment (georggrubner) [reply src/components/DataSetDropdownControl.js]: bfcaae3

File: src/components/visualizer/controls/UserControls/ProjectionControl.js (hunk 3 above)
Comment (georggrubner) [reply src/components/visualizer/controls/UserControls/ProjectionControl.js]: 2eca7dd

File: src/etc/const.js (hunk 4 above)
Comment (georggrubner) [reply src/etc/const.js]: dd50bb5

File: src/components/visualizer/controls/UserControls/ProjectionControl.js (hunk 3 above)
Comment (danielbartsch) [reply src/components/visualizer/controls/UserControls/ProjectionControl.js]: the parenthesis at the closing tag, too

**Pull request merged**